<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1480</width>
    <height>1052</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qMVG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLbl">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>1284</width>
      <height>484</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListView" name="ds_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>1021</height>
     </rect>
    </property>
   </widget>
   <widget class="QGLViewer" name="viewer" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>537</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="sampleBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>610</y>
      <width>711</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Robust Estimation</string>
    </property>
    <widget class="QFrame" name="ransacFrm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>671</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="lmedsFrm">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>71</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="goodLbl_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>threshold</string>
       </property>
      </widget>
      <widget class="QSlider" name="ransacThres">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>18</y>
         <width>121</width>
         <height>18</height>
        </rect>
       </property>
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QRadioButton" name="ransacBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>RANSAC</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="lmedsBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>24</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>LMEDS</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loadLImg">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>loadLeft</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadRImg">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>loadRight</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logMsgBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>799</y>
      <width>1281</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="vFeature">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>112</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>viewFeature</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vEssential">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>124</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>viewEssential</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vHomograph">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>0</y>
      <width>144</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>viewHomograph</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="featureBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Feature</string>
    </property>
    <widget class="QSlider" name="minDistRate">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>43</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="showLeft">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>35</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>L</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showRight">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>35</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showCrrsp">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>35</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="goodLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>37</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>minDistRate</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="dEssentia">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>0</y>
      <width>146</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>decmpEssential</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dHomograph">
    <property name="geometry">
     <rect>
      <x>936</x>
      <y>0</y>
      <width>170</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>decmpHomograph</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1480</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Dataset"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Dataset">
   <property name="text">
    <string>Open tum Dataset</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGLViewer</class>
   <extends>QWidget</extends>
   <header>QGLViewer/qglviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
